src/print.rs

```diff
@@ -49,7 +49,18 @@ pub fn prompt(args: ArgMatches) {
         write!(handle, "{}", module_without_prefix).unwrap()
     }
 
-    // Print all remaining modules
+    while let Some(module) = printable.next() {
+        write!(handle, "{}", module).unwrap();
+        if module.to_string().contains('\n') {
```

Comment:
I think a slightly more robust and declarative way to handle this would be to check the module's name, rather than its string representation. 🙂

Reply:
Thanks for your review. updated!

I'll squash commits later if you prefer that.